cf-plugin-backup-release.yaml

```diff
@@ -0,0 +1,120 @@
+shared:
```

Comment:
We don't need this separated out in this pipeline, since it's only used once (the master pipeline has like 5 of them).

Reply:
will do

Reply:
Done with commit 26ec7b3de6826d8677420fe18dc7d72b1f188c88